eagle-hadoop-metric/src/main/webapp/app/apps/hadoop_metric/ctrls/overview.js

```diff
@@ -21,52 +21,193 @@
 	 * `register` without params will load the module which using require
 	 */
 	register(function (hadoopMetricApp) {
-		hadoopMetricApp.controller("overviewCtrl", function ($scope, PageConfig) {
+		hadoopMetricApp.controller("overviewCtrl", function ($q, $wrapState, $scope, PageConfig, METRIC, Time) {
+			var cache = {};
+			var mapRes = common.map2;
+			$scope.site = $wrapState.param.siteId;
+			var activeMasterInfo = METRIC.hbaseActiveMaster($scope.site);
+			var metricMap = common.map1;
```

Comment:
* You can't modifiy `common` library for application usage.
* `map1` is magic name.